<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QDiagnosisTestBase</class>
 <widget class="QDialog" name="udg::QDiagnosisTestBase">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Diagnosis Tests</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="m_progressBarWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="m_animationInProgressLabel"/>
        </item>
        <item>
         <widget class="QLabel" name="m_executingTestLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="m_testsProgressBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_resultsWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="m_resultsIconLabel"/>
        </item>
        <item>
         <widget class="QLabel" name="m_resultsTitleLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Some tests have failed</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_viewTestsLabel">
        <property name="styleSheet">
         <string notr="true">#m_viewTestsLabel  {
	border-radius: 4px;
	border: 1px solid  #BBBBBB;
    background-color: #E4E4E4;
	padding: 5px;
	color: #333333;
}

#m_viewTestsLabel
{
	text-decoration: none;
}
</string>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;ok&quot; style=&quot;color:#333333&quot;&gt;View tests&lt;/a&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="m_buttonsAndScrollAreaWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="m_informationPushButton">
             <property name="text">
              <string>Show information (%1)</string>
             </property>
             <property name="icon">
              <iconset resource="../main/main.qrc">
               <normaloff>:/images/icons/emblem-information.svg</normaloff>:/images/icons/emblem-information.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_correctPushButton">
             <property name="text">
              <string>Show correct (%1)</string>
             </property>
             <property name="icon">
              <iconset resource="../main/main.qrc">
               <normaloff>:/images/icons/emblem-success.svg</normaloff>:/images/icons/emblem-success.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_warningsPushButton">
             <property name="text">
              <string>Show warnings (%1)</string>
             </property>
             <property name="icon">
              <iconset resource="../main/main.qrc">
               <normaloff>:/images/icons/emblem-warning.svg</normaloff>:/images/icons/emblem-warning.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="m_errorsPushButton">
             <property name="text">
              <string>Show errors (%1)</string>
             </property>
             <property name="icon">
              <iconset resource="../main/main.qrc">
               <normaloff>:/images/icons/emblem-error.svg</normaloff>:/images/icons/emblem-error.svg</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="m_diagnosisTestsResultsScrollArea">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="m_diagnosisTestsResults">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>756</width>
              <height>336</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="m_saveResultsButton">
          <property name="text">
           <string>Save Results</string>
          </property>
          <property name="icon">
           <iconset resource="../main/main.qrc">
            <normaloff>:/images/icons/document-save.svg</normaloff>:/images/icons/document-save.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>338</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="m_closeButton">
          <property name="text">
           <string>Close</string>
          </property>
          <property name="icon">
           <iconset resource="../main/main.qrc">
            <normaloff>:/images/icons/window-close.svg</normaloff>:/images/icons/window-close.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../main/main.qrc"/>
  <include location="../main/main.qrc"/>
  <include location="../main/main.qrc"/>
 </resources>
 <connections/>
</ui>
